### ISSUE TITLE
No matter how you try it, hiyacfw doesn't appear in unlaunch (i use translator)

### ISSUE THREAD

**Issue by taehyeon123** [mondul/HiyaCFW-Helper#89]
I'm successful, but if you check it with dsi, nothing happens. Can you help me

**Comment by mondul** [comment]
That's weird. Please check your SD card for errors, also try with a different one and let me know what happens

**Comment by Mayravixx** [comment]
Same thing here, but I think my issue is that I installed it to the copy I have on my PC, luckily for me, the original was still on the SD card, so I backed that up in a separate folder, and I'm currently trying it by using HiyaCFW on the nand on the SD card. One thing has changed, it's currently on "Extracting ARM7/ARM9 BIOS from NAND" but it has been there for a minute or two. The laptop I'm doing this on because it has an SD card slot, is a bit slow, only a 1.2 GHz CPU with 2 GB of DDR2 RAM so maybe it's got something to do with that? Will update if something changes.
